### PR TITLE
EntityBoardTest.java

### PR DESCRIPTION
**Pull request by 4stuntman4**
in the test, we insert correct values in the fields and save the form. The name of the form is Board. Verify that all values saved correctly. 
Not done - upload picture and verification of DateTime that has value 13/12/2020 18:02:42 and I have no idea how to compare it with minutes and sec - I always get different minutes and sec.

### CODE REVIEW

File: src/test/java/EntityBoardInputTest.java [hunk 1]
```diff
@@ -0,0 +1,172 @@
+
```
Comment (Stan575) [inline src/test/java/EntityBoardInputTest.java:1]: Delete empty lines

File: src/test/java/EntityBoardInputTest.java [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+import runner.BaseTest;
+import java.io.File;
+import java.io.IOException;
+import java.text.SimpleDateFormat;
+import java.util.Calendar;
+import java.util.List;
+import java.util.UUID;
+
+
+public class EntityBoardInputTest extends BaseTest {
```
Comment (Stan575) [inline src/test/java/EntityBoardInputTest.java:18]: Rename class to EntityBoardTest

File: src/test/java/EntityBoardInputTest.java [hunk 3]
```diff
@@ -0,0 +1,172 @@
...
+        WebElement createdRecordDate = driver.findElement(newRecordDate);
+        WebElement createdRecordDateTime = driver.findElement(newRecordDateTime);
+        WebElement createdRecordUser1Demo = driver.findElement(newRecordUser1Demo);
+
+        Assert.assertEquals(createdRecordText.getText(), text, "Created record text issue");
+        Assert.assertEquals(createdRecordStringPending.getText(), pending, "Created record Pending issue");
+        Assert.assertEquals(createdRecordInt.getText(), Integer.toString(number), "Created record number issue");
+        Assert.assertEquals(createdRecordDecimal.getText(), Double.toString(decimal), "Created record decimal issue");
+        Assert.assertEquals(createdRecordDate.getText(), currentDataEuropean, "Created date issue");
+        //Assert.assertEquals(createdRecordDateTime, currentDataTime, "Created dateTime issue"); // dodelat
```
Comment (Stan575) [inline src/test/java/EntityBoardInputTest.java:141]: Commented out code is prohibited per SergeiD

**Review by Stan575 — CHANGES_REQUESTED** [review]
Overall looks good, please do minor editions

File: src/test/java/EntityBoardInputTest.java (hunk 1 above)
Comment (4stuntman4) [reply src/test/java/EntityBoardInputTest.java:1]: deleted

File: src/test/java/EntityBoardInputTest.java (hunk 2 above)
Comment (4stuntman4) [reply src/test/java/EntityBoardInputTest.java:18]: renamed

File: src/test/java/EntityBoardInputTest.java (hunk 3 above)
Comment (4stuntman4) [reply src/test/java/EntityBoardInputTest.java:141]: deleted

**Comment by 4stuntman4** [comment]
closed due to unresolved issues

**Pull request closed without merging**